docs/components/gantry/oneaxis.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Configure a One-Axis Gantry"
+linkTitle: "oneaxis"
+weight: 70
+type: "docs"
+description: "Configure a oneaxis gantry."
+tags: ["gantry", "components"]
+# SME: Rand
+---
+
+Configure a `oneaxis` gantry to integrate a one-axis gantry into your robot:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter a name for your sensor, select the type `gantry`, and select the `oneaxis` model.
+
+Click **Create component**.
+Paste into the **Attributes** box:
+
+``` json
+"motor": <"your-motor-name">,
+"length_mm": <int>,
+"axis": {
+    "X": <int>,
+    "Y": <int>,
+    "Z": <int>
+}
+```
```

Comment:
```suggestion
```
This is unnecessary and could be confusing since all of this is autopopulated into the box by the app.